pkg/plugins/runtime/k8s/controllers/configmap_controller_test.go

```diff
@@ -201,7 +201,7 @@ var _ = Describe("ServiceToConfigMapMapper", func() {
 				},
 			},
 			[]kube_core.Pod{
-				podFn("pod1", map[string]string{"app": "app1"}, map[string]string{metadata.KumaMeshLabel: "mesh1"}),
+				podFn("pod1", map[string]string{"app": "app1"}, map[string]string{metadata.KumaMeshAnnotation: "mesh1"}),
```

Comment:
```suggestion
				podFn("pod1", map[string]string{"app": "app1"}, map[string]string{metadata.KumaMeshAnnotation: "mesh1"}), // nolint:staticcheck
```

right? instead of blanket ignoring it